src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -2642,23 +2747,27 @@ namespace WaterToAirHeatPumpSimple {
-                                                          SimpleWatertoAirHP(HPNum).RatedWaterVolFlowRate);
-            } else if (UtilityRoutines::SameString(SimpleWatertoAirHP(HPNum).WatertoAirHPType, "HEATING")) { // heating
-                coilSelectionReportObj->setCoilFinalSizes(SimpleWatertoAirHP(HPNum).Name,
-                                                          "Coil:" + SimpleWatertoAirHP(HPNum).WatertoAirHPType + ":WaterToAirHeatPump:EquationFit",
-                                                          SimpleWatertoAirHP(HPNum).RatedCapHeat,
-                                                          SimpleWatertoAirHP(HPNum).RatedCapHeat,
-                                                          SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate,
-                                                          SimpleWatertoAirHP(HPNum).RatedWaterVolFlowRate);
+        if (SimpleWatertoAirHP(HPNum).reportCoilFinalSizes) {
+            if (!DataGlobals::WarmupFlag && !DataGlobals::DoingHVACSizingSimulations && !DataGlobals::DoingSizing) {
+
+                if (UtilityRoutines::SameString(SimpleWatertoAirHP(HPNum).WatertoAirHPType, "COOLING")) { // cooling
+                    coilSelectionReportObj->setCoilFinalSizes(SimpleWatertoAirHP(HPNum).Name,
+                                                              "Coil:" + SimpleWatertoAirHP(HPNum).WatertoAirHPType +
+                                                                  ":WaterToAirHeatPump:EquationFit",
+                                                              SimpleWatertoAirHP(HPNum).RatedCapCoolTotal,
+                                                              SimpleWatertoAirHP(HPNum).RatedCapCoolSens,
+                                                              SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate,
+                                                              SimpleWatertoAirHP(HPNum).RatedWaterVolFlowRate);
+                } else if (UtilityRoutines::SameString(SimpleWatertoAirHP(HPNum).WatertoAirHPType, "HEATING")) { // heating
+                    coilSelectionReportObj->setCoilFinalSizes(SimpleWatertoAirHP(HPNum).Name,
+                                                              "Coil:" + SimpleWatertoAirHP(HPNum).WatertoAirHPType +
+                                                                  ":WaterToAirHeatPump:EquationFit",
+                                                              SimpleWatertoAirHP(HPNum).RatedCapHeat,
+                                                              SimpleWatertoAirHP(HPNum).RatedCapHeat,
+                                                              SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate,
+                                                              SimpleWatertoAirHP(HPNum).RatedWaterVolFlowRate);
+                }
+                SimpleWatertoAirHP(HPNum).reportCoilFinalSizes = false;
```

Comment:
This looks like it should go inside each if block otherwise the first coil will disable reporting for the second.

Reply:
No, this looks OK. HPNum should be unique for each coil type.